datasets/open_targets_genetics/pipelines/dataset.yaml

```diff
@@ -0,0 +1,30 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+dataset:
+  name: open_targets_genetics
+  friendly_name: ~
+  description: ~
+  dataset_sources: ~
+  terms_of_use: ~
+resources:
+
+  # - type: storage_bucket
+  #   name: "open-targets-genetics"
+  #   uniform_bucket_level_access: True
+  #   location: US
+
```

Comment:
You won't need these commented lines. You can remove them.

Reply:
@adlersantos done